components/HomeHeaderItems.vue

```diff
@@ -0,0 +1,223 @@
+<template>
+  <div class="swiperContainer">
+    <swiper
+      ref="homeSwiper"
+      class="swiper"
+      :options="swiperOptions"
+      @slideChange="onSlideChange"
+      @touchStart="onTouch"
+      @touchEnd="onTouch"
+    >
+      <swiper-slide v-for="item in items" :key="item.Id">
+        <div class="slide-backdrop">
+          <blurhash-image
+            :key="`${item.Id}-${reloadSentinel}`"
+            :item="getRelatedItem(item)"
+            :type="'Backdrop'"
+          />
+        </div>
+        <div class="slide-content">
+          <v-container
+            fill-height
+            class="mx-md-10 mt-md-5 py-0 py-md-4 align-end align-sm-center align-md-start"
+          >
+            <v-row>
+              <v-col cols="12" sm="8" md="6" xl="5" class="py-0 py-md-4">
+                <p class="text-truncate mb-2 my-2">
```

Comment:
```suggestion
                <p class="text-overline text-truncate mb-2 my-2">
```

This looks a bit better, I think.